app.py

```diff
@@ -1,10 +1,15 @@
-from flask import Flask
+from flask import Flask, render_template
+
 app = Flask(__name__)
 
+words = ["iso", "medo", "u", "ducan"]

```

Comment:
I want better words.

Reply:
Can we put these instead? 
"Pura" "Na" "Krovu" "Ikce" "Pikce"

Reply:
Yes.